### ISSUE TITLE
Startup Crash with Version 6.11 & 6.12

### ISSUE THREAD

**Issue by TheComputerizer** [tyra314/ToolProgression#73]
I recently updated from version 6.10 for my pack and am now getting [this crash](https://pastebin.com/6qw9ejQD). Also [here](https://pastebin.com/KNHCq6fQ) is the log associated with that crash. The error is talking about dynamic surrounding but I was able to confirm that it only crashes when this mod is updated

**Comment by tyra314** [comment]
That crash is caused by dynamic surroundings, not tool progression. Please report it there.

**Comment by TheComputerizer** [comment]
I know you said that the crash is not caused by tool progression, but the the dynamic surroundings dev pointed out that it is not actually caused by dynamic surrounding, but an incorrect ordering of dependencies that causes other stuff to fail. I dont know if I can pinpoint the exact error, but this crash definitely only started happening after updating this mod, so if you could at least take a look, I would appreciate it.

```
[18:07:51] [Client thread/ERROR]: The full error
net.minecraftforge.fml.common.toposort.ModSortingException: There was a cycle detected in the input graph, sorting is not possible
A dependency cycle was detected in the input mod set so an ordering cannot be determined
The first mod in the cycle is FMLMod:crafttweaker{4.1.20}
The mod cycle involves:
	FMLMod:p455w0rdslib{2.3.161} : before: [], after: [redstoneflux, mantle, tconstruct, enderio, projecte, tesla, thaumcraft, albedo]
	FMLMod:toolprogression{1.12.2-1.6.12} : before: [tconstruct, metallurgy, aoa3, twilightforest, brandoncore, codechickenlib], after: [gamestages, orestages]```

**Comment by tyra314** [comment]
I see. Use the old version. The newer release does nothing more than fiddling with dependencies.

**Comment by tyra314** [comment]
Can you try, if this build works:


[toolprogression-1.12.2-1.6.13-pre.zip](https://github.com/tyra314/ToolProgression/files/6338231/toolprogression-1.12.2-1.6.13-pre.zip)

**Comment by TheComputerizer** [comment]
Yeah sorry I wasnt able to test it today, I will do it when I wake up

**Comment by TheComputerizer** [comment]
It looks like I was able to start up just fine, thanks!

**Comment by TheComputerizer** [comment]
Is this build going to be published soon?

**Comment by TheComputerizer** [comment]
Hi, do you have any plans to push this change to curseforge?

**Comment by N-Montana** [comment]
> Can you try, if this build works:
> 
> [toolprogression-1.12.2-1.6.13-pre.zip](https://github.com/tyra314/ToolProgression/files/6338231/toolprogression-1.12.2-1.6.13-pre.zip)

I know this is a massive necro but I'm working on a modpack and was getting a crash with 6.10, .11 and .12 but that .13 fix is also solving whatever was happening to me. I would really appreciate it if you did release that fix to curse forge so that modpacks could use it, this mod makes life so much easier.
